gulpfile.js

```diff
@@ -232,6 +233,10 @@ function pxtcommon() {
     return Promise.resolve();
 }
 
+function notifyProcessingComplete() {
+    console.log("-- Processing Complete --");
```

Comment:
```suggestion
    console.log("-- Build Complete --");
```

maybe? 'Processing' sounds slightly off to me here but it doesn't really matter either way I suppose~